tests/Doctrine/Tests/Common/Annotations/AnnotationReaderTest.php

```diff
@@ -112,6 +113,21 @@ public function testPropertyAnnotationIsIgnored()
         self::assertEmpty($reader->getPropertyAnnotations($ref->getProperty('property')));
     }
 
+    /**
+     * @group 45
```

Comment:
not sure what this is for, but copied from the other ignored annotations tests.

Reply:
This is for the issue ID: this would be 244 in this case

Reply:
ah, that makes sense :-D 
fixing that number